.yamato/standalone-build-test.yml

```diff
@@ -15,6 +15,8 @@ test_mac_standalone_{{ editor.version }}:
     - pip install pyyaml
     - python -u -m ml-agents.tests.yamato.standalone_build_tests
```

Comment:
Maybe be explicit about 3DBall here (And in python-ll-api-test.yml)?

Reply:
The reason we do not do that is to be backward compatible with the old versions of the executable. @chriselion

Reply:
Oh right. Ugh.